project/BuildSettings.scala

```diff
@@ -60,6 +60,9 @@ object BuildSettings {
       "SNOWFLAKE_PRIVATE_KEY" -> "secretPrivateKey",
       "SNOWFLAKE_PRIVATE_KEY_PASSPHRASE" -> "secretKeyPassphrase",
       "HOSTNAME" -> "testWorkerId"
+    ),
+    libraryDependencySchemes ++= Seq(
+      "com.github.luben" % "zstd-jni" % VersionScheme.Always
```

Comment:
Why do we need this? Could you add a comment please?

Reply:
I think I hit this problem somewhere else, and I fixed it differently.

Bear with me while I try to find it, or jog my memory.

Reply:
I've added a comment to explain the issue.